lib/mayaUsd/ufe/UsdUndoCreateFromNodeDefCommand.cpp

```diff
@@ -51,20 +53,19 @@ Ufe::SceneItem::Ptr UsdUndoCreateFromNodeDefCommand::insertedChild() const
 
 void UsdUndoCreateFromNodeDefCommand::execute()
 {
+    UsdUndoBlock undoBlock(&_undoableItem);
     _addPrimCmd->execute();
     setIdAttr();
 }
 
 void UsdUndoCreateFromNodeDefCommand::undo()
 {
-    _addPrimCmd->undo();
-    // Nothing to do for the node:id attribute. It will get deleted with the prim.
+    _undoableItem.undo();
 }
```

Comment:
Did you run clang-format locally? I have a feeling it will put this entire method on a single line now.

Reply:
Good point, just uploaded a fix with the clang formatting.